### PR TITLE
added tutorials from pytorch library, adapted to whale dataset

### PR DESCRIPTION
**Pull request by James-Leslie**
Found two tutorials on PyTorch docs. I gave them each a run-through and made some minor changes to get it to work with our whale data.

Think we can build a more "development-ready" framework using a lot of the code. Run through the notebooks and see what you think.

### CODE REVIEW

**Comment by kieranbd** [comment]
Looks cool, thanks @James-Leslie. @devon12stone, if I recall correctly I may have changed batch size from 16 to 8 because I just couldn't get the training notebook to run initially.

File: 1_densenet.ipynb [hunk 1]
```diff
@@ -117,7 +117,7 @@
     "test = datasets.ImageFolder(data_dir+'/val', transform=test_transforms)\n",
     "\n",
     "trainloader = torch.utils.data.DataLoader(train, batch_size=16, shuffle=True, num_workers=4)\n",
-    "testloader = torch.utils.data.DataLoader(test, batch_size=8, shuffle=True, num_workers=4)"
+    "testloader = torch.utils.data.DataLoader(test, batch_size=16, shuffle=True, num_workers=4)"
```
Comment (James-Leslie) [inline 1_densenet.ipynb:120]: @devon12stone I changed `batch_size` here because the batch size of the testloader actually doesn't affect performance since there is no backpropagation. Increasing batch size speeds up training and inference, but too large and you can exceed your computer's memory. I have also noticed in the past that an increased batch size leads to a smoother training loss curve, but too large can flatten the curve out. It's almost like a bias-variance tradeoff scenario

**Comment by James-Leslie** [comment]
@devon12stone 
> ... do you think we would use fine tuning or feature extraction?
> 
I think we can try both and compare?
